docs/source/asr/speaker_recognition/configs.rst

```diff
@@ -1,32 +1,27 @@
 NeMo ASR Configuration Files
```

Comment:
Just noticed that this header is wrong, so links that try to go to the ASR configuration files documentation (rather than for speaker recognition, which is what's actually here) end up here instead.